src/plugins/data/server/config_deprecations.ts

```diff
@@ -8,17 +8,7 @@
 
 import type { ConfigDeprecationProvider } from '@kbn/core/server';
 
-export const autocompleteConfigDeprecationProvider: ConfigDeprecationProvider = ({
-  renameFromRoot,
-}) => [
-  renameFromRoot(
-    'kibana.autocompleteTerminateAfter',
```

Comment:
This was forgotten to remove in [this pr](https://github.com/elastic/kibana/pull/129977)